arch/arm/core/aarch64/Kconfig

```
@@ -58,6 +58,12 @@ config IPM_CONSOLE_STACK_SIZE
 
 if CPU_CORTEX_A
 
+config ARMV8_A_NS
+	bool "ARMv8-A Normal World (Non-Secure world of Trustzone)"
+	help
+	  This option signifies that Zephyr is entered in Trustzone
+	  Non-Secure state
```

Nit: the official capitalisation is `TrustZone`.
```suggestion
config ARMV8_A_NS
	bool "ARMv8-A Normal World (Non-Secure world of TrustZone)"
	help
	  This option signifies that Zephyr is entered in TrustZone
	  Non-Secure state
```